styles/Editor.css

```diff
@@ -939,6 +939,57 @@
   font-size: 75%;
 }
 
+.ProseMirror .table-footer  {
+  font-size: 14px;
+  padding-top: 10px;
+}
+
+.ProseMirror .table-footer .footnote-element:not(:first-child)::before {
+  content: '';
+  border-bottom: 1px dashed #C9C9C9;
+  display: block;
+  margin-bottom: 8px;
+}
+
+.ProseMirror .table-footer .footnote-element {
+  counter-reset: table_footnote;
+}
+
+.ProseMirror .table-footer .footnote-element p::before {
+  counter-increment: table_footnote;
+  content: counter(table_footnote);
+  display: inline-block;
+  position: relative;
+  cursor: pointer;
+  vertical-align: super;
+  font-size: 75%;
+  margin-right: 4px;
+}
+
+.ProseMirror .table-footer p {
+  margin-bottom: 0px;
+  margin-top: 0px;
+  padding: 4px;
+}
+
+.ProseMirror .table-footer .footnote-selected, .table-footer > div:has(> p.footnote-selected) > p  {
```

Comment:
This, along with the following styles responsible for implementing the selected state, appears to be drawn 😅. While it's a valid solution, there's an opportunity for a simpler and more refined implementation. I would recommend updating the table footer element's content DOM to wrap any paragraphs that represent general notes in a div and give it a class name 'general-note'. This will help simplify how you style it.

Reply:
As indicated in my comment at https://github.com/Atypon-OpenSource/manuscripts-body-editor/pull/179#issuecomment-1903499269, I plan to address the DOM update in the second phase. This is due to the complexities involved in wrapping the general footnotes , which is not as straightforward as other elements and will requite a different way for tracking the selected note than the current one. This approach is to save time, and facilitate its transition to QA before the end of the sprint/PI.

Reply:
As indicated in my comment at https://github.com/Atypon-OpenSource/manuscripts-body-editor/pull/179#issuecomment-1903499269, I plan to address the DOM update in the second phase. This is due to the complexities involved in wrapping the general footnotes , which is not as straightforward as other elements and will requite a different way for tracking the selected note than the current one. This approach is to save time, and facilitate its transition to QA before the end of the sprint/PI.